tidb-cloud/connect-tidbcloud-to-vercel.md

```diff
@@ -0,0 +1,110 @@
+
+The detailed steps are as follows:
+
+1. Click **Add Integration** in the upper-right area of the [TiDB Cloud Vercel Integration](https://vercel.com/integrations/tidb-cloud) page. The **Add TiDB Cloud** dialog displays.
+2. Select the scope of your integration in the drop-down list and click **CONTINUE**.
+3. Select the Vercel Projects to which the Integration will be added and click **CONTINUE**.
+4. Confirm the required permissions for integration and click **ADD INTEGRATION**. Then you are directed to an integration page of the TiDB Cloud console.
+5. On the integration page, select the target Vercel Projects, and select the target TiDB Cloud cluster after providing the cluster information. A TiDB Cloud cluster belongs to [an organizations and a project](/tidb-cloud/manage-user-access.md#view-the-organization-and-project).
+6. Click **Add Integration and Return to Vercel**.
+7. Back to your Vercel dashboard, go to your Vercel project, click **Settings** > **Environment Variables**, and confirm that the environment variables have been automatically added.
+
+    If the variables have been added, the connection is completed.
+
+After you have completed the integration setup and successfully connected a TiDB Cloud cluster to your Vercel projects, the information necessary for the connection is set in the projects' environment variables. The following are some common variables:
+
+```
+TIDB_HOST
+TIDB_PORT
+TIDB_USER
+TIDB_PASSWORD
+```
+
+For Dedicated Tier clusters, the root CA is set in this variable:
+
+```
+TIDB_SSL_CA
+```
+
+## Connect via manually setting environment variables
+
+1. Follow [connect to a TiDB Cloud cluster via standard connection](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) to get the connection information. Do not forget to set the **Allow Access from Anywhere** traffic filter on the **Security Settings** page and save the password.
+2. Go to your Vercel dashboard > Vercel project > **Settings** > **Environment Variables**, [declare each environment variable value](https://vercel.com/docs/concepts/projects/environment-variables#declare-an-environment-variable) according to the connection information in Step 1.
+
+The following is an example of the connection variables for a TiDB Cloud Dedicated Tier cluster:
+
+```
+var connection = mysql.createConnection({
+  host: '<your_host>',
+  port: 4000,
+  user: 'root',
+  password: '<your_password>',
+  database: 'test',
+  ssl: {
+    ca: fs.readFileSync('ca.pem'),
+    minVersion: 'TLSv1.2',
+    rejectUnauthorized: true
+  }
+});
+```
+
+In Vercel, you can declare the variables as follows. You can use whatever name that suits your projects.
+
+* **NAME** = TIDB\_HOST **VALUE** = `<your_host>`
+* **NAME** = TIDB\_PORT **VALUE** = 4000
+* **NAME** = TIDB\_USER **VALUE** = root
+* **NAME** = TIDB\_PASSWORD **VALUE** = `<your_password>`
+* **NAME** = TIDB\_SSL\_CA **VALUE** = `<content_of_ca.pem>`
+
+## Configure Connections
```

Comment:
```suggestion
## Modify connection settings
```

Reply:
I'd like to leave it as is. Since Vercel use the term "Configure".